bin/with_rubygems

```diff
@@ -3,28 +3,30 @@
 
 require "pathname"
 
-def run(*cmd)
-  return if system(*cmd)
-  raise "Running `#{cmd.join(" ")}` failed"
+module WithRubygems
+  def run(*cmd)
+    return if system(*cmd)
+    raise "Running `#{cmd.join(" ")}` failed"
+  end
+
+  module_function :run
 end
 
-version = ENV.delete("RGV")
+version = ENV.delete("RGV") || "2.7.7"
```

Comment:
I'm not sure I like this change -- it means I need to have the local tmp checkout of rubygems present to be able to use any of the development binstubs

Reply:
Yeah, it means that. Why is it a problem though? The checkout happens automatically so as a user of the binstubs nothing should change (other than a slightly slower first usage?). As I say in the commit message, I like it because it makes the specs more predictable and less dependent on the host system, and more similar to what's done in the CI.

Anyways, the reason I added this is that my alternative fix won't work without it, because when removing the entries from `LOAD_PATH` that include `bundler`, I'm also removing the entries that include `rubygems`. So unless `rubygems` is included in the `LOAD_PATH` separately, it doesn't work.

If you still don't want this, I can remove the change and make sure I always set `RGV` before running the specs, just let me know :+1:.

Reply:
It requires an internet connection to run the specs

Reply:
Yeah, the first time you run the specs. But the first time you run the specs also requires an internet connection in order to download bundler's repo.

Anyways, I'll remove the change!

Reply:
@segiddins I removed the commit! At this point I can also remove https://github.com/bundler/bundler/pull/6694/commits/e3395f44bceb2465fea882669f687bf678737377, since it's no longer necessary. Although it might be argued that it's a good change anyways, so I didn't (yet) remove it.

Reply:
By the way, I must've messed up something while running the specs, the commit author of the last commit is [lolwut](https://github.com/bundler/bundler/blob/368d7594adbdf83032719011d7608545faf2d942/spec/support/builders.rb#L655)! :fearful:

Reply:
I removed the mentioned commit and ammended the lolwut author! I got now a green local build :smile: :tada: